staging/src/k8s.io/kubectl/pkg/util/podutils/podutils.go

```diff
@@ -161,8 +168,14 @@ func (s ActivePods) Less(i, j int) bool {
 		return afterOrZero(podReadyTime(s[i]), podReadyTime(s[j]))
 	}
 	// 7. Pods with containers with higher restart counts < lower restart counts
-	if maxContainerRestarts(s[i]) != maxContainerRestarts(s[j]) {
-		return maxContainerRestarts(s[i]) > maxContainerRestarts(s[j])
+	regularRestartsI, sidecarRestartsI := maxContainerRestarts(s[i])
```

Comment:
Similar code appears multiple times in the PR.
Can this code be extracted into a helper ?

Reply:
Added a new function: compareMaxContainerRestarts